app/front-end/src/components/layouts/baseLayout.tsx

```diff
@@ -1,12 +1,6 @@
-import { IconTitleButton } from '@/components/buttons/IconTitleButton';
 import { SettingsDialog, ShortcutsDialog } from '@/components/dialogs';
+import { Sidebar } from '@/components/sidebar/sidebar';
```

Comment:
Add `index.ts` to remove redundancy in pathing

Reply:
Done!